native/run

```diff
@@ -3,16 +3,28 @@
 const { execSync } = require('child_process')
 const { program } = require('commander')
 
+function parseBashVariables(str) {
```

Comment:
I know this sounds bad but this what chat suggested to solve a problem that has to do with a long command in cmd (Error: spawnSync C:\WINDOWS\system32\cmd.exe ENAMETOOLONG
) when I run "start:malte" or "android:malte"